src/network/providers/ethereum.js

```diff
@@ -578,17 +591,26 @@ ethereum.getKeyValue = async (
         );
 
         const copyFromIkv_prolog = '@@copy_from_ikv=';
-        if (! value.startsWith(copyFromIkv_prolog)) {
+        if (!value.startsWith(copyFromIkv_prolog)) {
             return value; // no redirection
         } else {
             // the format is:
             // @@copy_from_ikv=<identity>:<key_in_ikv>
             const withoutPrefix = value.substring(copyFromIkv_prolog.length);
 
             const copy_identity = withoutPrefix.split(':')[0];
-            const copy_key = withoutPrefix.split(':').slice(1).join(':');
-
-            const value = await ethereum.getKeyValue(copy_identity, copy_key, 'latest', 'exact', true);
+            const copy_key = withoutPrefix
```

Comment:
Is there a way to avoid changes like that? I assume it was made automatically, right? I think in this case the changed string was less than 100 chars long. If it's true, then the change makes no sense and only complicates reviews.

Reply:
Yes, it was made automatically by running prettier as per the _.prettierrc.js_ config in the repo.

In my opinion, the best way to avoid is for everybody to run prettier on their changes, it's quite simple to set up "format on save" in any editor.

A second option would be to remove _.prettierrc.js_ and not use prettier at all.

A third option would be to leave things like they are and for me to turn off prettier locally. I just don't like to having a _.prettierrc.js_ and not using it. But I can do it if that's the preference.